modules/control-utility/src/test/java/org/apache/ignite/util/GridCommandHandlerTest.java

```diff
@@ -704,6 +704,77 @@ public void testClusterChangeTag() throws Exception {
         assertTrue("Tag has not been updated in 10 seconds", tagUpdated);
     }
 
+    /**
+     * Tests idle_verify working on an active cluster with persistence. Works via control.sh.
+     *
+     * @throws Exception if failed.
+     */
+    @Test
+    public void testIdleVerifyOnActiveClusterWithPersistence() throws Exception {
+        dataRegionConfiguration = new DataRegionConfiguration()
+                .setName("persistent-dataRegion")
+                .setPersistenceEnabled(true);
+
+        Ignite ignite = startGrids(1);
+
+        ignite.cluster().state(ACTIVE);
+
+        assertTrue(ignite.cluster().active());
```

Comment:
IMHO, this check is excessive.

Reply:
Thank you, I will remove that outside of try. Yes, it is for printing into idle_verify-[date].txt